.gitignore

```diff
@@ -75,6 +75,13 @@ src/api/ml/z3enums.ml
 src/api/ml/z3native.mli
 src/api/ml/z3enums.mli
 src/api/ml/z3.mllib
+src/api/js/node_modules/
```

Comment:
Not sure why they are in .gitignore.

Reply:
This is the usual practice for JS projects. This folder holds the dependencies used to build the project.